src/ai/ai.cpp

```diff
@@ -1536,9 +1536,8 @@ bool will_join_crisis_with_offer(sys::state& state, dcon::nation_id n, sys::cris
 	if(offer.target == state.world.nation_get_ai_rival(n))
 		return true;
 	auto offer_bits = state.world.cb_type_get_type_bits(offer.wargoal_type);
-	if((offer_bits & (military::cb_flag::po_demand_state | military::cb_flag::po_annex)) != 0)
+	if((offer_bits & (military::cb_flag::po_colony | military::cb_flag::po_demand_state | military::cb_flag::po_annex)) != 0)
```

Comment:
po_colony isn't place in the sun; that is also po_demand_state. po_colony is the special crisis cb that removes competitors from a potential colony IIRC. thus it can never appear in a third party offer in the first place